packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -32,7 +32,7 @@ import AlertBar from '../../components/AlertBar';
 import DialogMessage from '../../components/DialogMessage';
 import FetchErrorDialogMessage from '../../components/FetchErrorDialogMessage';
 import { LoadingOverlay } from '../../components/LoadingOverlay';
-import CloseIcon from '../../components/CloseIcon';
+import { ReactComponent as CloseIcon } from '@fxa-react/images/close.svg';
```

Comment:
Hm, I see this is being re-used and is being converted to a React Component, but the SVG itself is not really React-related. I have concerns about storing them in a dedicated directory like this for fear that @fxa-react becomes another junk drawer for tangentially related files. Could this SVG live adjacent to the components' files?

Reply:
If we had images located inside the component where they're being used, we wouldn't be able to cleanly reuse them, but I see your point about it not being "React". We were [exporting it as a React component](https://github.com/mozilla/fxa/pull/5318/files#diff-7efa0c29a1218ac2d1c78c1527dfce15L1-L2) (you may need to click "reveal" for `CloseIcon.tsx`) but if we did it that way, where would the SVG ultimately live if we need it in payments and settings? I guess we could also copy the contents of the SVG into an exported React component, but I personally prefer to keep the raw SVGs around. Let's rename to `fxa-shared-frontend`? :D (kidding, lol)

Reply:
Alright, approving on the back of your Slack chat about this being the best option. Let it be known that I'm only like... 90% on board, I still think we should find a better way to organize files that might be frontend-related but are not exclusive to React.